src/main/kotlin/com/machiav3lli/fdroid/database/entity/Repository.kt

```diff
@@ -501,6 +501,15 @@ data class Repository(
             "103063BC7189C91CE727DBF8266B07662518096E1686B6A088253933A3D0788F",
             ""
         )
+        private val DIVOLT = defaultRepository(
```

Comment:
Seems good, thanks!
Only for future additions: It should be optimally added to a `addedReposVX` as seen at the end of the file and to the auto migration of the DB version in `DatabaseX`.